docs/modules/ROOT/partials/recovery/recover-etcd.adoc

```diff
@@ -0,0 +1,147 @@
+[abstract]
+--
+ifeval::["{provider}" == "cloudscale"]
+Steps to recover etcd on an OpenShift 4 cluster on https://www.cloudscale.ch/[cloudscale.ch].
+endif::[]
+ifeval::["{provider}" == "exoscale"]
+Steps to recover etcd on an OpenShift 4 cluster on https://www.exoscale.com/[Exoscale].
+endif::[]
+--
+
+[WARNING]
+====
+Restoring to a previous cluster state is a destructive and destablizing action to take on a running cluster. This should only be used as a last resort.
+
+If you are able to retrieve data using the Kubernetes API server, then etcd is available and you shouldn't restore using an etcd backup.
+====
+
+== Starting situation
+
+ifeval::["{provider}" == "cloudscale"]
+* You have an OpenShift 4 cluster on cloudscale.ch
+endif::[]
+ifeval::["{provider}" == "exoscale"]
+* You have an OpenShift 4 cluster on Exoscale
+endif::[]
+* One of the following scenarios is true:
+** The cluster has lost the majority of control plane hosts (quorum loss).
+** An administrator has deleted something critical and must restore to recover the cluster.
```

Comment:
In this scenario, the cluster object backup should be sufficient in most cases, I think.

```suggestion
** An administrator has deleted a critical component which cannot be restored from the object backup.
```